internal/envoy/v3/route.go

```diff
@@ -402,3 +411,123 @@ func containsMatch(s string) *envoy_route_v3.HeaderMatcher_SafeRegexMatch {
 		SafeRegexMatch: SafeRegexMatch(regex),
 	}
 }
+
+func CookieRewriteConfig(policies []dag.CookieRewritePolicy) *any.Any {
+	codeTemplate := `
+function envoy_on_response(response_handle)
+	rewrite_table = {}
+
+	{{range $i, $p := .}}
+	function cookie_{{$i}}_attribute_rewrite(attributes)
+		response_handle:logDebug("rewriting cookie \"{{$p.Name}}\"")
+
+		{{if $p.Path}}attributes["Path"] = "Path={{$p.Path}}"{{end}}
+		{{if $p.Domain}}attributes["Domain"] = "Domain={{$p.Domain}}"{{end}}
+		{{if $p.SameSite}}attributes["SameSite"] = "SameSite={{$p.SameSite}}"{{end}}
+		{{if eq $p.Secure 1}}attributes["Secure"] = nil{{end}}
+		{{if eq $p.Secure 2}}attributes["Secure"] = "Secure"{{end}}
+	end
+	rewrite_table["{{$p.Name}}"] = cookie_{{$i}}_attribute_rewrite
+	{{end}}
+
+	function rewrite_cookie(original)
```

Comment:
Yeah that approach was what I wanted to do but the ability to use a script/module from the global filters wasn’t working for me

this will be refactored though, putting the rewrite data into per route filter data and using a common script to extract that and do rewrites

Reply:
I see you already mentioned this in the reviewer notes. Sorry for going through this too hastily!
(I recently started reviewing in vscode "github pull requests and issues" extension, instead of github.com and I'm bit disoriented where to find existing comments :-))